devtools/gradle/gradle-model/src/main/java/io/quarkus/gradle/tooling/ToolingUtils.java

```diff
@@ -38,21 +43,100 @@ public static boolean isEnforcedPlatform(ModuleDependency module) {
                 || Category.REGULAR_PLATFORM.equals(category.getName()));
     }
 
-    public static IncludedBuild includedBuild(final Project project,
-            final ProjectComponentIdentifier projectComponentIdentifier) {
-        final String name = projectComponentIdentifier.getBuild().getName();
+    public static IncludedBuild includedBuild(final Project project, final String buildName) {
+        Gradle currentGradle = project.getRootProject().getGradle();
+        while (null != currentGradle) {
```

Comment:
When can this happen?

Reply:
Calling `currentGradle.getParent()` at the bottom of the loop can return `null`, that's our exit condition for walking up the project tree.